sale_advance_payment/views/sale_view.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <button name="action_draft" position="after">
+                <button
+                    name="%(action_view_account_voucher_wizard)d"
+                    string="Pay sale advanced"
```

Comment:
I prefer Sale advance payment or may be  Register Sale Payment. It just a suggestion.